semantics.json

```diff
@@ -208,6 +208,15 @@
         "description": "Show points earned for each answer.",
         "importance": "low",
         "default": true
+      },
+      {
+        "name": "preventPointSubstraction",
+        "type": "boolean",
+        "label": "No score point deduction",
+        "importance": "low",
+        "description": "Enable to prevent score point deduction for wrong marked words (be careful: might be abused from users who simply mark all the words to get full score)",
+        "default": false,
+        "status": "new"
```

Comment:
Hmm, I don't think this went as it should have. `"status": "new"` should be gone automatically once H5P CLI has finished and and all the `"englishLabel"` and `"englishDefault"` values inside the language files should not be there.